pykokkos/interface/views.py

```diff
@@ -333,6 +333,10 @@ def _get_type(self, dtype: Union[DataType, type]) -> Optional[DataType]:
 
 
     def __eq__(self, other):
+        # Tempprary fix. Will change soon 
+        if other.dtype.type == np.float64:
+            return self.array == other
+
```

Comment:
Why/where is this needed?

Reply:
at a few places the Naive Bayes code creates subviews like:

```python3
X[ y == y_i, :]  # X: pk.View2D[pk.double], y: pk.View1D[pk.double], y_i: Union(np.float64, pk.double)
```

I have updated this to make it more generic and not restricted to `np.float64` only

Please let me know if this looks right.

Reply:
As discussed, `not hasattr(N, "__len__")` might make more sense here for now